components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa/src/Core/Extensions/X509CertificateChainModelEx.cs

```diff
@@ -75,7 +75,11 @@ public static class X509CertificateChainModelEx {
                 validator.ChainPolicy.RevocationFlag =
                     X509RevocationFlag.EntireChain;
                 validator.ChainPolicy.RevocationMode =
-                    X509RevocationMode.NoCheck;
+                    X509RevocationMode.Offline;
```

Comment:
Comment from @mregen: `X509RevocationMode.Offline` should not be used as there are strange particularities in the implementation, such as cert validator tries to reach a OCSP responder or CRL.

Reply:
see my comment below, NoCheck is the right choice here.